catalyst-uploader.go

```diff
@@ -27,8 +27,13 @@ func run() int {
 	// cmd line args
 	describe := flag.Bool("j", false, "Describe supported storage services in JSON format and exit")
 	timeout := flag.Duration("t", 30*time.Second, "Upload timeout")
+	vFlag := flag.Lookup("v")
+	verbosity := flag.String("v", "3", "Log verbosity. {4|5|6}")
+
 	flag.Parse()
 
+	vFlag.Value.Set(*verbosity)
+
```

Comment:
I don't know where do we spawn `catalyst-uploader` process but we should pass `-v` in there.